usagov/vote-gov-drupal
# Vote-736/style gov banner

## Jira ticket

[VOTE-736](https://cm-jira.usa.gov/browse/VOTE-736)

## Description

Style the official banner and language switcher for desktop and mobile.

## Deployment and testing

### Post-deploy

1. lando retune
2. lando drush uli
3. Add a sitewide alert

### QA/Test

1. At desktop dimensions, verify the banner and language switcher match the designs. Be sure to expand the banner.
2. Switch to mobile, verify the banner and language switcher match the designs.
3. Verify the site-wide alert displays below the banner (both closed and expanded)

## Checklist for the Developer




## Checklist for the Peer Reviewers
